tests/cases/standalone/common/flow/flow_flush.result

```diff
@@ -42,11 +42,8 @@ SELECT
 FROM
     out_num_cnt_basic;
 
-+---------------------------------+-------------------------+
-| sum(numbers_input_basic.number) | time_window             |
-+---------------------------------+-------------------------+
-| 42                              | 2021-07-01T00:00:00.100 |
-+---------------------------------+-------------------------+
+++
+++
```

Comment:
is this expected?

Reply:
> is this expected?

Currently this is expected, as data is inserted before flow's create, hence not possible to notify flow to flush correct time range, this can be refill later once we added `refill_flow` admin functions